src/paginate.ts

```diff
@@ -34,58 +35,40 @@ export async function paginateRaw<T>(
   const [page, limit, route] = resolveOptions(options);
 
   const totalQueryBuilder = queryBuilder.clone();
-  const items = await queryBuilder
-    .limit(limit)
-    .offset((page - 1) * limit)
-    .getRawMany<T>();
-
-  const total = await totalQueryBuilder.getCount();
+  const [items, total] = await Promise.all([
```

Comment:
:+1: 
Not sure why we're not using getManyAndCount, must've been an issue I can't remember?